app/views/govuk_publishing_components/components/docs/button.yml

```diff
@@ -98,4 +98,10 @@ examples:
     data:
       text: "Second button"
       inline_layout: true
+  with_name_and_value_set:
+    description: By default, the button has no value or name set so it will not pass information when the form is submitted. This allows a name and value to be added so a button can add information to the form submission.
```

Comment:
Have you got an example use case for needing this? Just curious!

Reply:
Um, not one that's live - just me trying to gracefully get a form to either display a filter page or a results page without losing the information either currently in the form or that's just been entered by the user.